rust/macros/get_id_info.rs

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: GPL-2.0
+
+use proc_macro::{TokenStream, TokenTree};
+
+pub(crate) fn get_id_info(ts: TokenStream) -> TokenStream {
```

Comment:
This could be a decl macro using `macro_rules!`. That is both simpler and gives you syntax highlighting.

Reply:
Done in the new commit.